cmd/gateway-conformance/main.go

```diff
@@ -129,20 +144,55 @@ func main() {
 						cmd.Env = append(cmd.Env, fmt.Sprintf("SUBDOMAIN_GATEWAY_URL=%s", subdomainGatewayURL))
 					}
 
-					cmd.Stdout = out{output}
+					cmd.Stdout = out{
+						Writer: output,
+						Filter: func(line string) bool {
+							return verbose || strings.Contains(line, "FAIL") || strings.HasPrefix(line, "PASS")
+						},
+					}
 					cmd.Stderr = os.Stderr
+
+					fmt.Println("Running tests...\n")
 					testErr := cmd.Run()
+					fmt.Println("\nDONE!\n")
+
+					if testErr != nil {
+						fmt.Println("\nLooking for details...\n")
+						strOutput := output.String()
+						lineDump := []string{}
+						for _, line := range strings.Split(strOutput, "\n") {
+							if strings.Contains(line, "FAIL") {
+								fmt.Println(line)
+								for _, l := range lineDump {
+									fmt.Println(l)
+								}
+								lineDump = []string{}
+							} else if strings.Contains(line, "===") {
```

Comment:
```suggestion
							} else if strings.HasPrefix(line, "=== ") {
```

should make that parsing more reliable

Reply:
Updated, we need to account for U+0016 though.